### PR TITLE
Decompile DRA func_80133FCC

### PR DESCRIPTION
**Pull request by cram0**
Should be good to go ?

### CODE REVIEW

**Comment by sozud** [comment]
I haven't checked this locally but I think it's more like this https://decomp.me/scratch/zc3Px

**Comment by cram0** [comment]
> I haven't checked this locally but I think it's more like this https://decomp.me/scratch/zc3Px

It is matching with the mentionned variable, thank you !

**Comment by sozud** [comment]
From what I remember D_800BD240 and D_800BD228 aren't real variables, they are offsets within g_XaMusicConfigs.

File: src/pc/stubs.c [hunk 1]
```diff
@@ -119,6 +119,9 @@ s32 D_800987B4;
 u8 g_PadsRepeatTimer[BUTTON_COUNT * PAD_COUNT];
 s32 D_80136410;
 s32 D_80136414[];
+s32 D_8013B65C;
+s32 D_800BD228;
```
Comment (sozud) [inline src/pc/stubs.c:123]:
```
g_XaMusicConfigs = 0x800BD224;
```
So use unk228 for D_800BD228 and  240 is probably the next element, by my count the struct is 16 bytes. So 0x800BD224+16+12 == g_XaMusicConfigs[pos + 1].unk22f?

Comment (cram0) [reply src/pc/stubs.c:123]: What I needed to do was to add 1 to the position to finally get it to match with the field `unk230` from that same structure, thanks !

**Review by sozud — APPROVED** [review]
Thanks for fixing the g_XaMusicConfigs thing

**Pull request merged**